myceliumd/Cargo.toml

```diff
@@ -15,7 +15,10 @@ path = "src/main.rs"
 clap = { version = "4.5.8", features = ["derive"] }
 tracing = { version = "0.1.40", features = ["release_max_level_debug"] }
 tracing-logfmt = "0.3.4"
-tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
+tracing-subscriber = { version = "0.3.18", features = [
+  "env-filter",
+  "nu-ansi-term",
+] }
```

Comment:
Let's add the corresponding change to myceliumd-private as well. While we currently don't install it, users could still manually compile it